io/eolearn/io/sentinelhub_service.py

```diff
@@ -6,9 +6,10 @@
 import datetime as dt
 
 import numpy as np
+from eolearn.core import EOPatch, EOTask, FeatureType, get_common_timestamps
```

Comment:
Why changing order of imports? In general imports from the same package (or the same namespace package in this case) are written last.

Reply:
Because of linter. Honestly, I don't care.